plugins/backend/catalog-backend-module-okta/README.md

```diff
@@ -44,10 +60,10 @@ The provider allows configuring Okta search filtering for users and groups. See
 catalog:
   providers:
     okta:
-      - orgUrl: 'https://tenant.okta.com'
-        token: ${OKTA_TOKEN}
-        userFilter: profile.department eq "engineering"
-        groupFilter: profile.name eq "Everyone"
+      orgUrl: 'https://tenant.okta.com'
```

Comment:
Does this mean we can only configure one okta provider? Surely the array was better?

Reply:
No, I updated the [documentation for having multiple account configs](https://github.com/RoadieHQ/roadie-backstage-plugins/blob/5957568ae41ae665d28a0e96089d4c7be61ac80b/plugins/backend/catalog-backend-module-okta/README.md?plain=1#L22-L35). Having an object allows to have proper naming of the catalog location, which is needed to have a scheduler per config.
If it's not under a provider id will use the `default`. It is being done similarly in other catalog providers:
- [catalog-backend-module-msgraph](https://github.com/backstage/backstage/blob/4913addc6531826eee1a6756cef83bb63d3d48b6/plugins/catalog-backend-module-msgraph/src/microsoftGraph/config.ts)
- [catalog-backend-module-github](https://github.com/backstage/backstage/blob/8960902a4ea002d0a44c376f071048b0ef90053f/plugins/catalog-backend-module-github/src/providers/GithubEntityProviderConfig.ts)